hypothesis-python/src/hypothesis/extra/cli.py

```diff
@@ -82,6 +82,14 @@ def main():
     def obj_name(s: str) -> object:
         """This "type" imports whatever object is named by a dotted string."""
         s = s.strip()
+        if "/" in s:
```

Comment:
Don't forget about backslashes for Windows users!

Reply:
Good point! ⚡